client/src/common/components/CardItem.scss

```diff
@@ -31,6 +31,8 @@
     svg {
       height: 12px;
       width: 12px;
+      min-height: 12px;
+      min-width: 12px;
```

Comment:
is this necessary when height and width are 'hard set' above??

Reply:
Yes, it's necessary. Without these rules, if you add a long list title, the ellipsis is added but lock icon is very small (both in Chrome and Firefox):
![Firefox_Screenshot_2019-04-23T05-37-32 262Z](https://user-images.githubusercontent.com/25374390/56557264-b35b8800-659a-11e9-9a51-28f3344a17dc.png)